<launch>
  <include file="$(find plating_demo)/launch/realsense_camera_rgb.launch"/>

  <node name="publish_food_location" pkg="food_perception" type="publish_food_location" output="screen">
    <param name="camera_topic" value="/camera/color/image_raw"/>
    <param name="table_frame_id" value="base_link"/>
  </node>
</launch>
